tests/pfc/test_unknown_mac.py

```diff
@@ -164,8 +164,6 @@ def populateArp(unknownMacSetup, flushArpFdb, ptfhost, duthosts, rand_one_dut_ho
     # Wait 5 seconds for secondary ARP before proceeding to clear FDB
     time.sleep(5)
 
-    yield
```

Comment:
Can you elaborate a bit more on this random ip flush? remove_ip.sh script should clean up ip from all interfaces so that should clear the ip from the interface we used for the test as well. Is this ip already assigned to some other interface? if that's case, it might be a residue of some other tests, I would suggest to do a cleanup in the beginning as well as the end so that this test exits clean

Reply:
The IP is used by the interface we randomly picked up in the test. It is normal. What I fix here is the IP should be cleaned at the beginning of the test case, as the interface will respond the ARP request from DUT if the IP is assigned to this interface, then the DUT can learn the MAC of this interface. In that condition, this test case will fail.

Reply:
as discussed offline, let's assume that dut arp request would be within the 5sec delay that we have specified. If not, will need to increase the delay.